Automattic/wp-calypso
# Introduce scroll buttons to reader discover navigation.

Related to #

## Proposed Changes

* Introduces scroll tabs to the discover navigation.  These should be hidden to screen readers, keyboard navigation, and hidden when keyboard nav is in use - as these buttons do not make sense for screen reader and keyboard nav flows. They should also not display at the corresponding ends of scroll position:

![discover-nav-final](https://github.com/Automattic/wp-calypso/assets/28742426/0dff0c65-b33e-45f4-a253-b74e44408332)

## Testing Instructions



* Run this calypso branch locally.
* navigate to the discover page.
* verify the new scroll buttons appear on the top navigation tabs, test them.
* they should not appear when at their respective edges.
* they should not be part of the tab index, and should not be visible when keyboard navigation is in use.

## Pre-merge Checklist
